### PR TITLE
Add properties tab to metadata

### PR DESCRIPTION
**Pull request by penk85**
WPC-422: Add tab in metadata on articles and pages which gives allows setting key value properties.

### CODE REVIEW

**Review by bishoymelek — DISMISSED** [review]
:tada:

**Review by HanaAyman — APPROVED** [review]

File: packages/editor/src/client/panel/articleMetadataPanel.tsx [hunk 1]
```diff
@@ -384,6 +410,48 @@ export function ArticleMetadataPanel({
             />
           </Panel>
         )
+      case MetaDataType.Properties:
+        return (
+          <Panel>
+            <ControlLabel>{t('articleEditor.panels.properties')}</ControlLabel>
+            <ListInput
```
Comment (roosnic1) [inline packages/editor/src/client/panel/articleMetadataPanel.tsx:417]: I think we should also add a little info text here similar to the social media metadata. It should read something like "Properties can be set per article/page. Public properties are accessible through the public API"

**Review by roosnic1 — CHANGES_REQUESTED** [review]
very good work just some minor changes requested.

**Review by roosnic1 — APPROVED** [review]
🎉

**Review by HanaAyman — APPROVED** [review]

**Pull request merged**